src/common/context/forms/forms.test.tsx

```diff
@@ -1,12 +1,24 @@
 import { act, renderHook } from "@testing-library/react-hooks";
 import React, { FunctionComponent } from "react";
 import { FormsContextProvider, useFormsContext } from "./index";
+import { useConfigContext } from "../config";
+
+jest.mock("../config");
+
+const mockUseConfigContext = useConfigContext as jest.Mock;
 
 const wrapper: FunctionComponent = ({ children }) => (
   <FormsContextProvider>{children}</FormsContextProvider>
 );
 
 describe("useFormsContext", () => {
+  beforeAll(() => {
+    mockUseConfigContext.mockReturnValue({
+      config: {
+        forms: [{ schema: "FORM_1_SCHEMA" }, { defaults: { foo: "bar" }, schema: "FORM_2_SCHEMA" }],
```

Comment:
suggestion to rename: defaults => defaultValues

defaults is a bit generic and might become eventually problematic to understand if we need to introduce any other default related parameter

Reply:
hmm i dont necessary agree with this one. I'm indifferent to either, just have to change all the config file to use it. If someone wants to do it out of this PR I wont object.

![image](https://user-images.githubusercontent.com/7406870/89157090-828c4000-d59e-11ea-965d-da5f72516039.png)